docs/versions.md

```diff
@@ -1,6 +1,6 @@
 # Upgrading Velero versions
 
-Velero supports multiple concurrent versions. Whether you're setting up Velero for the first time or upgrading to a new version, you need to pay careful attention to versioning. This doc page is new as of version 0.10.0, and will be updated with information about subsequent releases.
+Velero supports multiple concurrent versions. Whether you're setting up Velero for the first time or upgrading to a new version, you need to pay careful attention to versioning.
```

Comment:
Reading this paragraph, I'm not 100% sure what the first sentence means, especially as we move into 1.0. Does it mean that the Velero team supports multiple concurrent versions? Or that users can run multiple concurrent versions in the cluster?

Reply:
It means the former. Should I just delete this page?

Reply:
That works for me; I'm not seeing a whole lot of unique information here.

Reply:
Yeah, I think the intention was to describe breaking changes across pre-1.0 versions, but that shouldn't really be a concern going fwd. And we've added other notes to tell people to use official released versions and version-tagged docs. I'll go ahead and remove.

Reply:
deleted